grpc-proto/proto/query.proto

```diff
@@ -488,8 +488,8 @@ message ResultSet {
   // The rows of CQL data.
   repeated Row rows = 2;
 
-  // The number of rows.
-  google.protobuf.Int32Value page_size = 3;
+  // Deprecated: result sets don't include this field anymore. It can be inferred from rows.size.
+  google.protobuf.Int32Value page_size = 3 [deprecated = true];
```

Comment:
I'm not sure how strict we want to be with backward compatibility at this stage.

Alternatively, I can remove the field entirely, and maybe even adjust the index of the next field...

Reply:
No backward compatibility. We should just remove it.